docs/tables/scaleway_account_project.md

```diff
@@ -0,0 +1,19 @@
+# Table: scaleway_account_project
+
+A Project is a grouping of Scaleway resources. Each Scaleway Organization comes with a default Project, and you can create new Projects if necessary. Projects are cross-region, meaning resources located in different regions can be grouped in one single Project. When grouping resources into different Projects, you can use IAM to define custom access rights for each Project.
+
+This table requires an Organization ID to be configured in the scaleway.spc file.
+
+## Examples
```

Comment:
Are there any additional examples you'd find useful to add here? What queries are you running with this table in your env?

Reply:
The table is mainly used to feed other tables that need the project list.